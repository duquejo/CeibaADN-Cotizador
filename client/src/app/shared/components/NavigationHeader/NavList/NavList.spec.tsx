import * as React from 'react';
import { ShallowWrapper, mount, render, shallow } from 'enzyme';
import { NavList } from './index';

describe('Pruebas sobre componente <NavList />', () => {

    let componentWrapper: ShallowWrapper;
    let componentProps: React.ComponentProps<typeof NavList>;

    beforeEach( () => {
        componentProps = {
            items: [{
                label: 'Item menú 1',
                url: '/menu1'
            },{
                label: 'Item menú 2',
                url: '/menu2'
            },{
                label: 'Item menú 2',
                url: '/menu2'
            }
        ]};
        componentWrapper = shallow(<NavList { ...componentProps } />);
    });

    it('Debería renderizar el elemento', () => {
        expect( componentWrapper ).toMatchSnapshot();
    });
});