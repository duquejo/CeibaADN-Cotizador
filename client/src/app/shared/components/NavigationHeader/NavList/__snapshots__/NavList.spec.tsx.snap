// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas sobre componente <NavList /> Debería renderizar el elemento 1`] = `
<styled.ul>
  <li
    key="Item menú 1"
  >
    <NavItem
      label="Item menú 1"
      to="/menu1"
    />
  </li>
  <li
    key="Item menú 2"
  >
    <NavItem
      label="Item menú 2"
      to="/menu2"
    />
  </li>
  <li
    key="Item menú 2"
  >
    <NavItem
      label="Item menú 2"
      to="/menu2"
    />
  </li>
</styled.ul>
`;
